frontend/webadmin/modules/gwt-common/src/main/resources/org/ovirt/engine/ui/common/CommonApplicationConstants_ru.properties

```diff
@@ -20,6 +20,7 @@ addInstanceImages=\u0421\u043E\u0437\u0434\u0430\u0442\u044C
 addItemButtonLabel=\u0414\u043E\u0431\u0430\u0432\u0438\u0442\u044C
 addPermission=\u0414\u043E\u0431\u0430\u0432\u0438\u0442\u044C
 addSanStorage=\u0414\u043E\u0431\u0430\u0432\u0438\u0442\u044C
+addSystemPermission=\u0414\u043e\u0431\u0430\u0432\u0438\u0442\u044c \u0441\u0438\u0441\u0442\u0435\u043c\u043d\u044b\u0445 \u0440\u0430\u0437\u0440\u0435\u0448\u0435\u043d\u0438\u0439
```

Comment:
Should probably be `\u0414\u043e\u0431\u0430\u0432\u0438\u0442\u044c \u0441\u0438\u0441\u0442\u0435\u043c\u043d\u044b\u0435 \u0440\u0430\u0437\u0440\u0435\u0448\u0435\u043d\u0438\u044f`?
(2 characters changed)...

Reply:
You right... I thought about 2 variants, but for some reason wrote some not correct mid-version.

Reply:
Great, can close this comment :)